OpenEdXMobile/src/main/java/org/edx/mobile/view/LoginActivity.java

```diff
@@ -170,10 +166,14 @@ protected void onRestoreInstanceState(Bundle savedInstanceState) {
     }
 
     @Override
-    protected void onActivityResult(int requestCode, int resultCode, Intent data) {
+    public void onActivityResult(int requestCode, int resultCode, Intent data) {
         super.onActivityResult(requestCode, resultCode, data);
         tryToSetUIInteraction(true);
         socialLoginDelegate.onActivityResult(requestCode, resultCode, data);
+
+        if (resultCode == RESULT_OK) {
+            showAlertDialog(getString(R.string.success_dialog_title_help), getString(R.string.success_dialog_message_help));
```

Comment:
This is still not done in your latest commit (2d9f17599c62d0aedd07c38ecb88c45d8b52e867).